src/senaite/core/configure.zcml

```diff
@@ -16,7 +16,6 @@
   <include package=".catalog" />
   <include package=".datamanagers" />
   <include package=".exportimport" />
-  <include package=".locales" />
```

Comment:
Is this inteded?

Reply:
Oops, you're right, the locales directory must be registered in zcml before it is picked up on Zope startup

Reply:
O well, the locales is already registered here https://github.com/senaite/senaite.core/blob/address-dx/src/senaite/core/configure.zcml#L11 . This was only an import of locales as a python package because the COUNTRIES, allong with getGeoStates, etc. was living inside there. So translations should work without problems, yes